### PR TITLE
gnome-extra/gnome-commander: version bump and remove old

### PR DESCRIPTION
**Pull request by turboscholz**

### CODE REVIEW

File: gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild [hunk 1]
```diff
@@ -31,6 +31,9 @@ RDEPEND="
 	pdf? ( >=app-text/poppler-0.18 )
 	taglib? ( >=media-libs/taglib-1.4 )
 "
+
+PATCHES=( "${FILESDIR}/${P}-exiv2-0.27.1-missing-header.patch" )
```
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: what difference is there to the `1.10.0` patch?

Comment (turboscholz) [reply gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: There is actually no difference. But to be consistent, I added the same patch in a new file. Should I hard-code the file name to the original patch of version 1.10.0?

Comment (a17r) [reply gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: yes please, we don't duplicate patches.

File: gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild [hunk 2]
```diff
@@ -1,18 +1,18 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
```
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: any chance for an EAPI-7 bump here?

File: gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild (hunk 1 above)
Comment (turboscholz) [reply gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: done

File: gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild (hunk 2 above)
Comment (turboscholz) [reply gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: Thanks for pointing that out. But let me postpone the EAPI-bump to the next version-bump of gnome-commander.

File: gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild [hunk 3]
```diff
@@ -31,6 +31,9 @@ RDEPEND="
 	pdf? ( >=app-text/poppler-0.18 )
 	taglib? ( >=media-libs/taglib-1.4 )
 "
+
+PATCHES=( "${FILESDIR}/gnome-commander-1.10.0-exiv2-0.27.1-missing-header.patch" )
```
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild]: well, you could have used `${PN}-1.10.0` instead of `${P}`

**Pull request closed without merging**